wondrous-app/components/Table/MembershipRequests.tsx

```diff
@@ -145,6 +88,7 @@ export function MembershipRequestTable(props) {
                       }}
                     />
                     <Link passHref href={`/organization/${request?.orgUsername}/boards`}>
+                      {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
```

Comment:
If there is no clear solution yet, we may leave the error instead of disabling it completely. We can return to this if we find a better solution.

Reply:
yes let's try not to disable unless necessary